voxelbotutils/cogs/utils/custom_context.py

```diff
@@ -134,3 +134,14 @@ async def send(
             if ignore_error:
                 return None
             raise e
+            
+    async def reply(content:str, *args, embed:discord.Embed=None, ignore_error:bool=False, **kwargs):
```

Comment:
You're missing `self` from this, as well as `content` needing to be defaulted to `None`.